### PR TITLE
[Update] Readme Update

### PR DESCRIPTION
**Pull request by ckyiricky**
Add usage and interface documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,3 +11,20 @@ Kubernetes. This specific project will include:
 * Writing a report that summarizes the different approaches that can be used for
   topology aware routing along with a recommendation.
 
+## Usage
+go run main.go -input=inputFile -output=outputFile -alg=algorithm
+
+example of intput file (csv): each zone with number of nodes first, number of endpoints next  
+input name, zone1, zone2, zone3  
+perfect input, 10 10, 10 10, 20 20
```
Comment (robscott) [inline README.md:20]: The example input files I've seen don't have spaces after the commas, just wanted to confirm that this also works.

Comment (robscott) [inline README.md:20]: This should be a code block - add lines before and after with 3 backticks (```)

**Review by robscott — COMMENTED** [review]
Thanks for the work on this!

File: README.md (hunk 1 above)
Comment (ckyiricky) [reply README.md:20]: yeah, this works as well.

Comment (ckyiricky) [reply README.md:20]: fixed

**Review by robscott — APPROVED** [review]
LGTM, thanks!

**Pull request merged**